tests/test_dimension.py

```diff
@@ -369,7 +369,7 @@ def test_subdimleft_notparallel(self):
         assert all(i.is_Affine and i.is_Sequential for i in iterations if i.dim == xl)
         assert all(i.is_Affine and i.is_Parallel for i in iterations if i.dim == yi)
 
-        op.apply(time_m=0, time_M=0)
+        op.apply(time_m=1, time_M=1)
```

Comment:
what happened here?

Reply:
Modulo change. Mainly the stencil is `u[t+1,...] = u[t+1]``` with 0th time_order so first `time` to be updated is `t+1` for `t=0` so `1`.